cvrf2csaf/cvrf2csaf.py

```diff
@@ -59,39 +59,29 @@ def __init__(self, config):
         self.product_tree = ProductTree()
         self.vulnerability = Vulnerability()
 
+        # ToDo: Lang and SourceLang are missing here.
+        self.sections_handlers = {
+            'Acknowledgements': self.document_acknowledgments,
+            'AggregateSeverity': self.document_aggregate_severity,
+            'DocumentType': self.document_category,
+            'DocumentDistribution': self.document_distribution,
+            'DocumentNotes': self.document_notes,
+            'DocumentPublisher': self.document_publisher,
+            'DocumentReferences': self.document_references,
+            'DocumentTitle': self.document_title,
+            'DocumentTracking': self.document_tracking,
+            'ProductTree': self.product_tree,
+            'Vulnerability': self.vulnerability,
+        }
```

Comment:
@tschmidtb51, @sthagen: Here is the direct assignment you wished for (no more if/elif statement here). :)